mne/io/base.py

```diff
@@ -1558,13 +1558,13 @@ def plot(self, events=None, duration=10.0, start=0.0, n_channels=20,
              highpass=None, lowpass=None, filtorder=4, clipping=None,
              show_first_samp=False, proj=True, group_by='type',
              butterfly=False, decim='auto', noise_cov=None, event_id=None,
-             show_scrollbars=True, verbose=None):
+             show_scrollbars=True, use_scalebars=True, verbose=None):
```

Comment:
I think "show_scalebars" is more consistent with how we name other parameters, and is more specific too ("use" is a rather generic verb)

Reply:
Wanted to keep the wording similar. So I use the "use_scalebars" because in _plot_raw_traces, the parameter is named "use_scalebars". Should I still make this change?